src/dotnet/project.json

```diff
@@ -54,7 +54,7 @@
         "Microsoft.NETCore.ConsoleHost": "1.0.0-rc2-23811",
         "Microsoft.NETCore.TestHost": "1.0.0-rc2-23811",
         "NETStandard.Library": "1.0.0-rc2-23811",
-        "System.Reflection.Metadata": "1.2.0-rc3-23811",
+        "System.Reflection.Metadata": "1.3.0-beta-23811",
```

Comment:
It doesn't need to be. You can use project model stand alone so it doesn't matter. This was legit breaking Roslyn in dotnet, that's why I updated it